src/vs/workbench/contrib/chat/common/languageModelToolsService.ts

```diff
@@ -29,13 +30,20 @@ interface IToolEntry {
 	impl?: IToolImpl;
 }
 
+export interface IToolInvokationDto {
```

Comment:
Maybe it is just naming but the concepts of `dto` should be encapsulated inside extHost/mainThreadXYZ. In our services we always pretent that there is no EH/IPC